common/app/views/support/HtmlCleaner.scala

```diff
@@ -337,19 +337,16 @@ case class TagLinker(article: Article)(implicit val edition: Edition, implicit v
     s"""(.*)( |^)($tagName)( |,|$$|$dot|$question)(.*)""".r
   }
 
-  private def or[A](pred: A => Boolean, pred2: A => Boolean)(a: A) = pred(a) || pred2(a)
-
   def clean(doc: Document): Document = {
 
     if (article.content.showInRelated) {
 
       // Get all paragraphs which are not contained in a pullquote or in an instagram caption
-      val paragraphs = doc.getElementsByTag("p").filterNot( p => p.parents.exists(
-          or((ancestor: Element) =>
-            ancestor.tagName() == "aside" && ancestor.hasClass("element-pullquote"),
-            _.hasClass("instagram-media")
-          )
-        )
+      val paragraphs = doc.getElementsByTag("p").filterNot( p => p.parents.exists( ancestor => {
+          val inPullquote = ancestor.tagName() == "aside" && ancestor.hasClass("element-pullquote")
+          val inInstagramBlock = ancestor.hasClass("instagram-media")
+          inPullquote || inInstagramBlock
+        })
```

Comment:
Regarding the curly braces, you would usually make the function passed to `exists` a block by itself (rather than a single expression (ie: **{** thing => expression **}** vs thing => **{** expression **}**
You would get something like this:
````
p.parents.exists( { ancestor => 
   val inPullquote = ancestor.tagName() == "aside" && ancestor.hasClass("element-pullquote")
   val inInstagramBlock = ancestor.hasClass("instagram-media")
   inPullquote || inInstagramBlock
} )
```

Then you would be able abbreviate and to omit the unnecessary parenthesis:
```
p.parents.exists { ancestor => 
   val inPullquote = ancestor.tagName() == "aside" && ancestor.hasClass("element-pullquote")
   val inInstagramBlock = ancestor.hasClass("instagram-media")
   inPullquote || inInstagramBlock
}
```